sweep/sweeper.go

```diff
@@ -125,6 +206,21 @@ func (p *pendingInput) parameters() Params {
 	return p.params
 }
 
+// terminated returns a boolean indicating whether the input has reached a
+// final state.
+func (p *pendingInput) terminated() bool {
```

Comment:
suggestion: rename this `isTerminalState` or `isTerminal`

Reply:
i guess you also hate names like `initializable`, `updatable` or `removable`🤓

Reply:
I guess this is coming from a place where boolean predicates often start with `is`. I'm not gonna die on that hill though. As long as we have a consistent approach I can learn a new convention :)